ci/test.sh

```diff
@@ -154,6 +159,21 @@ $root/ci/test/resources_memcache.sh &
 $root/ci/test/app_internal_communication.sh &
 wait
 
+# import/export test
+convox apps create httpd-export
+convox deploy -a httpd-export --manifest convox-export.yml
```

Comment:
I'm using a new manifest since Redis and Memcached cannot be exported.
MariaDB and Mysql are not working either(also in previous versions). I'll create a ticket to address this problem in the future.